server/src/com/cloud/storage/StorageManagerImpl.java

```diff
@@ -1736,7 +1737,10 @@ public boolean storagePoolHasEnoughSpace(List<Volume> volumes, StoragePool pool)
                     allocatedSizeWithtemplate = _capacityMgr.getAllocatedPoolCapacity(poolVO, tmpl);
                 }
             }
-            if (volumeVO.getState() != Volume.State.Ready) {
+            // A ready state volume is already allocated in a pool. so the asking size is zero for it.
+            // In case the volume is moving across pools or is not ready yet, the asking size has to be computed
+            s_logger.debug("pool id for the volume with id: " + volumeVO.getId() + " is: " + volumeVO.getPoolId());
```

Comment:
@wido numeric ids in logs are easy to understand/debug than uuids. names might me more readable but ids are unique and hence no ambiguity. 
The current pool-id is already logged and the pool-id of the the volume would help me see if the volume is moving across pools. That was the information I was looking for while debugging this issue and was hard to find given nothing was logged before.